### ISSUE TITLE
Unable to detect gpu PCI ID on intel iris xe

### ISSUE THREAD

**Issue by Florennum** [benjamimgois/goverlay#96]
Issue: gooverlay not detecting GPU PCI ID for mangohud

Hello there!

Issue Details:
I'm currently facing an issue with gooverlay while trying to use mangohud. The problem is that gooverlay isn't detecting or finding my GPU's PCI ID automatically. I have to resort to manually finding it using `lspci | grep VGA` and then setting it in the config.

Expected Behavior:
Automatically detects my gpu PCI ID.

Hardware:
CPU: 11th Gen Intel i5-1135G7
GPU: Intel TigerLake-LP GT2 [Iris Xe Graphics]
Laptop: Lenovo V15 G2 ITL

Additional Information:
Installed mangohud and gooverlay with pacman.
Using Arch Linux.

Screenshot:
![image](https://github.com/benjamimgois/goverlay/assets/95132822/4cc6140c-52a1-4d85-8a33-630c6ccabab7)

I appreciate any assistance or guidance on resolving this issue. Thank you!

**Comment by benjamimgois** [comment]
Hi @Florennum , could you post the full output of your "lspci" command ?

**Comment by Florennum** [comment]
> Hi @Florennum , could you post the full output of your "lspci" command ?

yeah here it is in [pastebin](https://pastebin.com/sjvrVJmR) because something is wrong with formatting

**Comment by benjamimgois** [comment]
Take a look at 1.1 release. It might be fixed
